lgc/patch/FragColorExport.cpp

```diff
@@ -29,8 +29,10 @@
  ***********************************************************************************************************************
  */
 #include "lgc/patch/FragColorExport.h"
+#include "lgc/BuilderBase.h"
 #include "lgc/LgcContext.h"
 #include "lgc/patch/Patch.h"
+#include "lgc/patch/VertexFetch.h"
```

Comment:
Why do you need VertexFetch.h?

Reply:
I need to access `VertexFetch::getVgprTy`

Reply:
Could we move that somewhere more common then? Such as a function in Internal.h?

Reply:
Done.